diff -Naur lxqt-session-0.14.1/startlxqt.in lxqt-session-0.14.1.tpg/startlxqt.in
--- lxqt-session-0.14.1/startlxqt.in	2019-02-26 01:22:39.000000000 +0000
+++ lxqt-session-0.14.1.tpg/startlxqt.in	2020-03-09 13:32:23.471010150 +0000
@@ -6,6 +6,14 @@
     [ "$str" = "$substr" -o -z "${str##$substr:*}" -o -z "${str##*:$substr:*}" -o -z "${str%%*:$substr}" ]
 }
 
+# (tpg) create localized user dirs
+if $(command -v xdg-user-dirs-update); then
+    xdg-user-dirs-update
+fi
+
+# (cb) remove the ksycoca cache which could possibly have been invalid
+rm -f $HOME/.cache/ksycoca* ||:
+
 if [ -z "$XDG_DATA_HOME" ]; then
     export XDG_DATA_HOME="$HOME/.local/share"
 fi
@@ -23,6 +31,7 @@
 fi
 export XDG_DATA_DIRS
 
+export XDG_CONFIG_DIRS=/etc/xdg
 if [ -z "$XDG_CONFIG_DIRS" ]; then
     export XDG_CONFIG_DIRS="@PREDEF_XDG_CONFIG_DIRS@"
 else
@@ -95,5 +104,62 @@
 
 export XDG_CURRENT_DESKTOP="LXQt"
 
+# (tpg) copy default settings because LxQt cannot read them from /etc
+if test "x$XDG_CONFIG_HOME" = "x"
+then
+    BASEDIR="$HOME/.config"
+else
+    BASEDIR="$XDG_CONFIG_HOME"
+fi
+
+if [ ! -d $BASEDIR/lxqt ]; then
+    mkdir -p $BASEDIR/lxqt
+    cp -af /usr/share/lxqt/*.conf $BASEDIR/lxqt
+fi
+
+if [ ! -d $BASEDIR/pcmanfm-qt ]; then
+    mkdir -p $BASEDIR/pcmanfm-qt/lxqt
+    cp -af /usr/share/pcmanfm-qt/lxqt/*.conf $BASEDIR/pcmanfm-qt/lxqt
+fi
+
+if [ ! -e $XDG_CONFIG_HOME/lxqt.firstboot ]; then
+#(tpg) add special icons on DESKTOP
+    USER_DESKTOP="$(xdg-user-dir DESKTOP)"
+
+    if [ ! -e $USER_DESKTOP/om-welcome.desktop ]; then
+	if [ -e /usr/share/applications/om-welcome.desktop ]; then
+	    cp -f /usr/share/applications/om-welcome.desktop $USER_DESKTOP 2> /dev/null
+	    chmod +x $USER_DESKTOP/om-welcome.desktop
+	fi
+    fi
+
+    if [ ! -e $USER_DESKTOP/join.desktop ]; then
+	cp -f /usr/share/applications/join.desktop $USER_DESKTOP 2> /dev/null
+	chmod +x $USER_DESKTOP/join.desktop
+    fi
+
+    if [ ! -e $USER_DESKTOP/donate.desktop ]; then
+	cp -f /usr/share/applications/donate.desktop $USER_DESKTOP 2> /dev/null
+	chmod +x $USER_DESKTOP/donate.desktop
+    fi
+
+    if [ ! -e $USER_DESKTOP/calamares.desktop ]; then
+	if [ -e /usr/share/applications/calamares.desktop ]; then
+	    cp -f /usr/share/applications/calamares.desktop $USER_DESKTOP 2> /dev/null
+	    chmod +x $USER_DESKTOP/calamares.desktop
+	fi
+    fi
+
+    touch $XDG_CONFIG_HOME/lxqt.firstboot
+
+fi
+
+# workaround for xfwm4 not being start on first boot
+if cat $BASEDIR/lxqt/session.conf | grep -w window_manager=xfwm4 > /dev/null 2>&1; then
+    if ! pgrep "xfwm4" > /dev/null 2>&1; then
+	exec /usr/bin/xfwm4 &
+    fi
+fi
+
 # Start the LXQt session
 exec lxqt-session
