diff -up lxqt-session-2.1.0/config/lxqt.conf.4~ lxqt-session-2.1.0/config/lxqt.conf
--- lxqt-session-2.1.0/config/lxqt.conf.4~	2024-11-05 19:28:19.603621142 +0100
+++ lxqt-session-2.1.0/config/lxqt.conf	2024-11-05 19:30:52.365306241 +0100
@@ -1,12 +1,12 @@
 [General]
 theme=Valendas
 icon_theme=breeze
-single_click_activate=false
+single_click_activate=true
 tool_button_style=ToolButtonTextBesideIcon
 palette_override=true
 
 [Qt]
-font="Sans,11,-1,5,50,0,0,0,0,0"
+font="Noto Sans,11,-1,5,400,0,0,0,0,0,0,0,0,0,0,1"
 doubleClickInterval=400
-style=Fusion
+style=Breeze
 wheelScrollLines=3
