diff -up lxqt-session-1.1.0/config/lxqt.conf.1~ lxqt-session-1.1.0/config/lxqt.conf
--- lxqt-session-1.1.0/config/lxqt.conf.1~	2022-04-23 20:00:17.710102246 +0200
+++ lxqt-session-1.1.0/config/lxqt.conf	2022-04-23 20:02:22.265752111 +0200
@@ -1,5 +1,5 @@
 [General]
-theme=Clearlooks
+theme=Kde-plasma
 icon_theme=breeze
 single_click_activate=false
 tool_button_style=ToolButtonTextBesideIcon
diff -up lxqt-session-1.1.0/config/session.conf.1~ lxqt-session-1.1.0/config/session.conf
--- lxqt-session-1.1.0/config/session.conf.1~	2022-04-16 15:16:20.000000000 +0200
+++ lxqt-session-1.1.0/config/session.conf	2022-04-23 20:00:17.710102246 +0200
@@ -1,13 +1,16 @@
 [General]
 leave_confirmation=true
+window_manager=openbox
 
 [Environment]
 GTK_CSD=0
 GTK_OVERLAY_SCROLLING=0
+BROWSER=/usr/bin/falkon
+TERM=qterminal
 
 [Mouse]
 cursor_size=18
-cursor_theme=whiteglass
+cursor_theme=breeze_cursors
 acc_factor=20
 acc_threshold=10
 left_handed=false
