diff -up lxqt-session-2.4.0/lxqt-config-session/x11settings.cpp.3~ lxqt-session-2.4.0/lxqt-config-session/x11settings.cpp
--- lxqt-session-2.4.0/lxqt-config-session/x11settings.cpp.3~	2026-04-20 00:00:00.000000000 +0000
+++ lxqt-session-2.4.0/lxqt-config-session/x11settings.cpp	2026-04-20 00:00:00.000000000 +0000
@@ -33,7 +33,7 @@ static const QLatin1String windowManager
 static const QLatin1String QtScaleKey("QT_SCALE_FACTOR");
 static const QLatin1String x11LockCommandKey("lock_command");
 static const QLatin1String GdkScaleKey("GDK_SCALE");
-static const QLatin1String openboxValue("openbox");
+static const QLatin1String openboxValue("/usr/bin/openbox");
 
 X11Settings::X11Settings(LXQt::Settings *settings, QWidget *parent) :
     QWidget(parent),
